### PR TITLE
Move stuff out of 'skeleton' module and clean up Django settings

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: xenserver/settings.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+def abspath(*args):
+    """convert relative paths to absolute paths relative to BASE_DIR"""
+    return os.path.join(BASE_DIR, *args)
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '02qbonw5tuu$m(c^s9h5u+ravid-i+@kn(^5^aw!@o^nl*+!9c'
```
Comment (jerith) [inline xenserver/settings.py]: Can we make this invalid and thus require it to be provided at deploy time?

File: xenserver/settings.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (jerith) [inline xenserver/settings.py]: Can we default this to `False`?

File: xenserver/urls.py [hunk 3]
```diff
@@ -1,9 +1,6 @@
 from django.conf.urls import patterns, include, url
-from django.views.generic import RedirectView
-
-# Uncomment the next two lines to enable the admin:
 from django.contrib import admin
-admin.autodiscover()
```
Comment (jerith) [inline xenserver/urls.py]: Do we not need this?

Comment (JayH5) [inline xenserver/urls.py]: It's not needed in Django 1.7+ (it's now automatic)

File: xenserver/settings.py (hunk 2 above)
Comment (JayH5) [inline xenserver/settings.py]: It's tricky to change this and the `SECRET_KEY` to be required without making the dev setup more tricky... But I'll see what I can do.

File: xenserver/settings.py [hunk 4]
```diff
@@ -15,10 +15,10 @@ def abspath(*args):
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '02qbonw5tuu$m(c^s9h5u+ravid-i+@kn(^5^aw!@o^nl*+!9c'
+SECRET_KEY = None
```
Comment (jerith) [inline xenserver/settings.py]: Maybe add a comment explaining why this is `None`?

**Pull request merged**